google/cloud/forseti/services/scanner/dao.py

```diff
@@ -53,16 +55,17 @@ class Violation(base):
 
         __tablename__ = violations_tablename
 
+        full_name = Column(String(1024))
```

Comment:
not a big deal, but it's nicer to see the key column `id` as the first one, and every other column can be sorted after.

Reply:
Done.